test/graphql/types/Fund/updater.test.ts

```
@@ -0,0 +1,257 @@
+import { beforeEach, describe, expect, it, vi } from "vitest";
+import type { GraphQLContext } from "../../../../src/graphql/context";
+import type { FastifyBaseLogger } from "fastify";
+import { User } from "~/src/graphql/types/User/User";
```

Biome (1.9.4)

[error] All these imports are only used as types.

Importing the types with import type ensures that they are removed by the compilers and avoids loading unnecessary modules.
Safe fix: Use import type.


(lint/style/useImportType)